src/nvim/popupmenu.c

```diff
@@ -785,36 +787,41 @@ void pum_redraw(void)
   }
 }
 
-/// set info text to preview buffer.
+/// Set the informational text in the preview buffer when the completion
+/// item does not include a dedicated preview or popup window.
+///
+/// @param[in]  buf        Pointer to the buffer (buf_T) where the text will be set.
+/// @param[in]  info       Informational text to display in the preview buffer.
+/// @param[in]  lnum       Pointer to a line number (linenr_T), indicating where to start the text.
```

Comment:
don't repeat the types in the param descriptions

also, this is a pointer so the more interesting info is that it's updated. a hint about how it's updated would be useful.

I am just guessing about the "incremented for each..." part. please confirm it.

```suggestion
/// @param[in]  lnum       Where to start the text. Incremented for each added line.
```